### PR TITLE
FIX some typos

### PR DESCRIPTION
**Pull request by wulymammoth**

### CODE REVIEW

**Comment by ConsenSys-Academy** [comment]
Hi there! some of these have been fixed, would you pull from the main repo, merge those with your local branch and commit It should look like the following:

git remote add upstream https://github.com/ConsenSys-Academy/Blockchain-Developer-Bootcamp
git fetch upstream
git merge upstream/typo
git push origin typo

**Comment by cooganb** [comment]
Hey! Lmk if you'd like to try and fix this, otherwise I'll close in 24hrs

**Comment by wulymammoth** [comment]
@cooganb fixed 👍

File: docs/S01-fundamentals/M2-consensus/L5-trustless-consensus/index.html [hunk 1]
```diff
@@ -94,7 +93,11 @@ <h3>Periods and Proof of Work</h3>
       </ul>
 
       <p>
-        The following section will describe the process of creating a block and how blocks chained together create an immutable ledger that all network participants can validate on their own. This individual validation, without having to rely on trust, is how proof of work consensus maintains state in a distributed and safe way. 
+<<<<<<< HEAD
+        The following section will describe the process of creating a block and how blocks chained together create an immutable ledger that all network participants can validate on their own. This individual validation, without having to rely on trust, is how proof of work consensus maintains state in a distributed and safe way.
```
Comment (ConsenSys-Academy) [inline docs/S01-fundamentals/M2-consensus/L5-trustless-consensus/index.html:97]: Hey! I think this is an issue with the conflict resolution?

Comment (wulymammoth) [reply docs/S01-fundamentals/M2-consensus/L5-trustless-consensus/index.html:96]: removed conflict market

**Pull request closed without merging**